pkgs/tools/networking/connman/connman/default.nix

```diff
@@ -73,7 +73,7 @@ stdenv.mkDerivation rec {
     # Fix Musl build by avoiding a Glibc-only API.
     (fetchpatch {
       url = "https://git.alpinelinux.org/aports/plain/community/connman/libresolv.patch?id=e393ea84386878cbde3cccadd36a30396e357d1e";
```

Comment:
We also started using fetchurl instead of fetchpatch for these, since they are not generated but static files checked into the alpine repository.

Reply:
Good point, I'll change it.